openreview/tools.py

```diff
@@ -547,6 +551,11 @@ def subdomains(domain):
     domain_components = [c for c in full_domain.split('.') if c and not c.isspace()]
     domains = ['.'.join(domain_components[index:len(domain_components)]) for index, path in enumerate(domain_components)]
     valid_domains = [d for d in domains if not tld.is_tld(d)]
+
+    for d in valid_domains:
+        if d in duplicate_domains:
+            valid_domains.append(duplicate_domains[domain])
```

Comment:
Should it be like this?
```suggestion
            valid_domains.append(duplicate_domains[d])
```

Reply:
Done.